common/static/common/js/spec/components/paging_spec.js

```diff
@@ -68,30 +66,25 @@ define([ "jquery", "js/common_helpers/ajax_helpers", "URI",
             var pagingView;
 
             beforeEach(function () {
-                var assets = new AssetCollection();
-                assets.url = "assets_url";
-                var feedbackTpl = readFixtures('system-feedback.underscore');
```

Comment:
It used to be needed because the PagingView subclasses NotificationView which attempts to find the template when it initialized. As part of moving it out to common, I removed that dependency because it was never actually used.